src/main/scala/temple/generate/target/openapi/OpenAPIGenerator.scala

```diff
@@ -103,7 +104,28 @@ private class OpenAPIGenerator private (name: String, version: String, descripti
               500 -> Response.Ref(useError(500)),
             ),
           )
-      case Read   => // TODO in future PR
+      case Read =>
+        path(s"/$lowerName/{id}") += HTTPVerb.Get -> Handler(
+            s"Look up a single $lowerName",
+            tags = tags,
+            parameters = Seq(
+              Parameter(
+                InPath,
+                name = "id",
+                required = Some(true),
+                schema = OpenAPISimpleType("number", "int32"),
```

Comment:
I _believe_ this has gone to `string` (format `uuid`) now, but can do that in a follow up 👍